docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,171 @@ description: Next.js supports built-in image optimization, as well as third part
+## Using the Image Component
 
-Instead of optimizing images at build time, Next.js optimizes images on-demand, as users request them. Unlike static site generators and static-only solutions, your build times aren't increased, whether shipping 10 images or 10 million images.
+To add an image to your application, import the [`next/image`](/docs/api-reference/next/image.md) component:
 
-Images are lazy loaded by default. That means your page speed isn't penalized for images outside the viewport. Images load as they are scrolled into viewport.
+```jsx
+import Image from 'next/image'
+```
 
-Images are always rendered in such a way as to avoid [Cumulative Layout Shift](https://web.dev/cls/), a [Core Web Vital](https://web.dev/vitals/) that Google [uses in search ranking](https://developers.google.com/search/blog/2020/05/evaluating-page-experience).
+Now, you can define the `src` for your image (either local or remote).
 
-## Image Component
+### Local Images
 
-To add an image to your application, import the [`next/image`](/docs/api-reference/next/image.md) component:
+To use a local image, `import` your `.jpg`, `.png`, or `.webp` files:
 
 ```jsx
+import profilePic from '../public/me.png'
+```
+
+Dynamic `await import()` or `require()` are _not_ supported.
+
+Next.js will automatically determine the `width` and `height` or your image based on the imported file. These values are used to prevent [Cumulative Layout Shift](https://nextjs.org/learn/seo/web-performance/cls) while your image is loading.
+
+```js
 import Image from 'next/image'
+import profilePic from '../public/me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src="/me.png"
+        src={profilePic}
         alt="Picture of the author"
-        width={500}
-        height={500}
+        // width={500} automatically provided
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // placeholder="blur" // Optional blur-up while loading
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
-
-export default Home
 ```
 
-## Image Imports
-
-You can statically `import` images that live in your project. Dynamic `await import()` or `require()` are _not_ supported.
+### Remote Images
 
-With static `import`s, you only need to provide the `src` prop. The `width`, `height`, and `blurDataURL` props will automatically be populated. Alt text is still needed separately.
+To use a remote image, the `src` property should be a URL string, which can be [relative](#loaders) or [absolute](#domains). Because Next.js does not have access to remote files during the build proccess, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and optional [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually:
```

Comment:
@atcastle I think "relative" is incorrect.

Normal `<img>` elements support "relative" URLs such as `./img.png` or `../img.png` which are relative to the current page, but Next.js does not.

Next.js only supports absolute URLs, either full URL or implicit domain name as described here:

https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_URL#examples_of_absolute_urls

Reply:
I suppose you could say that the `src` is relative to the `loader` so maybe this is okay 🤔

Reply:
Right I just meant you can enter a relative URL as the src, and the loader will expand it into an actual URL. I guess maybe relative isn't the right word, because you can't use "../", for instance. I can't really think of a more appropriate word, though.

Reply:
Typo: proccess => process